src/Faker/Provider/ne_NP/Address.php

```diff
@@ -62,33 +62,7 @@ class Address extends \Faker\Provider\Address
         'Yala', 'Yalbang', 'Yamphodin', 'Yrsa',
     );
 
-    protected static $country = array(
```

Comment:
Just to let you know because, 
https://github.com/fzaninotto/Faker/blob/master/src/Faker/Provider/en_NZ/Address.php#LC53
has a single country.